packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -69,54 +76,42 @@ export const getPatientByDemo = async ({
   facilityId: string;
   cxId: string;
   demo: PatientData;
-}): Promise<Patient | null> => {
-  const { log } = Util.out(`getPatientByDemo - cxId ${cxId}`);
+}): Promise<Patient | undefined> => {
+  // TODO this normalization should not be rollled out until we do a migration to normalize all existing patients.
+  // A null normalizaed patient means that the patient demographics included default values.
+  const normalizedPatientDemo = normalizePatientDataMPI(convertPatientDataToPatientDataMPI(demo));
+  if (!normalizedPatientDemo) return undefined;
 
```

Comment:
We should think about our approach here. The Incoming patient's demographics are normalised and inside matchPatients the demographics of the patients we are matching against are normalised. Normalisation is like arguably the single highest value step for increasing patient matching percentages from the research I read. 

Considerations are: 
1. the existing way we store demographics
2. how to deal with default values. Part of most normalisation schemes is to remove "default" values that aren't really demographic data. i.e. a patient whose address is 123 main street or phone number is 000-000-0000. It pollutes the overall data pool.

Reply:
@jonahkaye is this comment still relevant after yesterday's convo?

Reply:
This is resolvable for the current PR, but I feel like we probably need to have a brief conversation about normalization to get on the same page enough to create a ticket to address down the line